src/App.tsx

```diff
@@ -1,14 +1,85 @@
 /* eslint-disable max-len */
```

Comment:
Consider removing the max-len eslint rule and configure the line length limit to a reasonable value instead of disabling it.